tools/packaging/kata-deploy/helm-chart/kata-deploy/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: "3.10.0"
+version: "3.10.1"
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "3.10.0"
+appVersion: "3.10.1"
```

Comment:
This is a good reminder to work on the PR that can update this via release process so that we only update one file. Single source of truth @zvonkok